src/state/inviteMemberProject/inviteMemberProjectSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+export const inviteMemberPrject = createAsyncThunk(
+  "project/invite",
+  async (member, { rejectWithValue }) => {
+    try {
+      const response = await fetch(
+        "https://sq-project-service.herokuapp.com/project/invite",
```

Comment:
Not like this. Check `config` file. Import `api` from there, and use that like:
`{api}/project/invite`